performance.py

```diff
@@ -371,30 +426,46 @@ async def main(
         "--num-instances",
         type=int,
         default=2,
-        help="The number of nodes to be tested",
+        help="The number of nodes for test",
     )
     parser.add_argument(
         "--running-time",
         default=1200,
         type=int,
         help=(
-            "How long should the instance run before the cluster is stopped "
+            "Option to set time the instance run before the cluster is stopped"
             "(in seconds, default: %(default)s)"
         ),
     )
+    parser.add_argument(
+        "--transfer",
+        default=False,
```

Comment:
You can use `action='store_true'` to turn it into a flag

https://docs.python.org/3/library/argparse.html#action